maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -186,6 +195,31 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					}
 					platformMode = false;
 					break;
+				case O3_Distribution:
+					wizard.promptForO3RefAppVersionIfMissing(server, versionsHelper);
+					Artifact artifact = new Artifact(server.getDistroArtifactId(), server.getVersion(),
+							server.getDistroGroupId(), "zip");
+					Properties frontendProperties;
+					if (server.getVersion().equals(versionsHelper.getLatestReleasedVersion(artifact))) {
```

Comment:
Would this be the latest released version? Or latest snapshot version?

Reply:
What it should be??

Reply:
I was only asking to know which is which, according to you?

Reply:
Does the main branch of https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/frontend/spa-build-config.json contains versions for the latest released version or the latest snapshot version?

Reply:
Releases are in tags. The main branch has the latest snapshot version.